packages/auto_mappr/lib/src/extensions/dart_object_extension.dart

```diff
@@ -20,7 +20,7 @@ extension DartObjectExtension on DartObject {
     final asFunction = toFunctionValue();
     if (asFunction != null) {
       return EmitterHelper.current.refer(asFunction.referCallString, asFunction.library.identifier).call([
-        if (passModelArgument) refer('model'),
+        if (passModelArgument && asFunction.parameters.isNotEmpty) refer('model'),
```

Comment:
Maybe we should rename `passModelArgument` to `maybePassModelArgument` or something that would suggest it might still not pass it in the end?